libsoftwarecontainer/src/container.cpp

```diff
@@ -622,12 +625,23 @@ bool Container::bindMountInContainer(const std::string &pathInHost,
                 log_error() << "Could not create file " << tempPath;
                 return false;
             }
-            m_cleanupHandlers.push_back(new FileCleanUpHandler(tempPath));
```

Comment:
Why is this line removed? If I read this correctly, this was the cleanup handler for a file that was touched in case we didn't mount to a dir but to a file. So why not cleanup that file just as we clean up the directories? Especially in the case where things go bad.